shell/utils/formatter.js

```diff
@@ -0,0 +1,12 @@
+
+const OPAQUE_SECRET_TYPE = 'Opaque';
```

Comment:
Please consider using the `./shell/config` for static configuration.
On fact for this case it seems to exist already [`/shell/config/secret.js`](https://github.com/rancher/dashboard/blob/master/shell/config/secret.js)